### PR TITLE
New settings icon - replacement?

### PR DESCRIPTION
**Pull request by delnyn**
Hello, just adding this file because I personally don't really like the original settings icon. I used SVGOMG, changed the color to "currentColor" and put all metadata in the <svg> tag. This is how it looks
image
and the 100% scale 
settings-2

### CODE REVIEW

**Comment by mittalyashu** [comment]
After optimising

```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M11.967 2h-.434a2 2 0 00-2 2v.18a2 2 0 01-.999 1.731l-.435.252a2 2 0 01-2.003 0l-.144-.084a2 2 0 00-2.735.733l-.22.383a2 2 0 00.73 2.73l.155.088a2 2 0 01.998 1.732v.51a2 2 0 01-.998 1.731l-.154.09a2 2 0 00-.732 2.73l.221.382a2.001 2.001 0 002.735.733l.144-.084a2 2 0 012.003 0l.435.252a2 2 0 01.999 1.731V20a2 2 0 002 2h.434a2 2 0 002-2v-.18a2 2 0 01.999-1.731l.435-.252a2 2 0 012.003 0l.144.084a2.002 2.002 0 002.735-.733l.22-.383a2 2 0 00-.73-2.73l-.155-.088a2 2 0 01-.998-1.732v-.51a2 2 0 01.998-1.731l.154-.09a2 2 0 00.732-2.73l-.221-.382a1.999 1.999 0 00-2.735-.733l-.144.084a2 2 0 01-2.003 0l-.435-.252a2 2 0 01-.999-1.731V4a2 2 0 00-2-2v0z" />
  <path d="M11.75 14a2.25 2.25 0 100-4.5 2.25 2.25 0 000 4.5z" />
</svg>

```

**Comment by delnyn** [comment]
I already optimized it.

**Comment by mittalyashu** [comment]
@delnyn The icon should have `path` tag, not circle.

**Comment by ericfennis** [comment]
@delnyn @mittalyashu I think this icon is too similar with the current settings icon. So I'm more for a replacement instead of a second one.

Can you maybe post an a side-by-side image with the two settings icons?

**Comment by mittalyashu** [comment]
@ericfennis After looking at this icon I think this looks more cleaner with less curvy lines and it will even look better when `stroke-width` is increased.

**Comment by delnyn** [comment]
image

**Comment by ericfennis** [comment]
I agree with you @mittalyashu this icon is much cleaner. Also better for lower resolution screens.
I think the icon look a bit better by increasing the circle in the middle, but I don't know what you think.
image
@delnyn I noticed that the code of the icon, the icon is a bit shifted to left. You can also see that in the preview you posted.

**Comment by delnyn** [comment]
> I think the icon look a bit better by increasing the circle in the middle, but I don't know what you think.
Yes, I had that idea too!
> I noticed that the code of the icon, the icon is a bit shifted to left. You can also see that in the preview you posted.
Actually it's because the outside path is 20h x 19.5w and that's pretty hard to deal with, I'll try to change that

**Comment by delnyn** [comment]
I made the icon centered, the circle 6px wide, removed settings-2 tags, made the attributes cleaner, and replaced settings with settings-2

**Comment by mittalyashu** [comment]
To decrease the size of first node, I tried to split into 4 `path`, it didn't work.

CleanShot 2021-05-01 at 11 44 57@2x

File: icons/settings.svg [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+<path d="M12.217 2h-.434a2 2 0 00-2 2v.18a2 2 0 01-.999 1.731l-.435.252a2 2 0 01-2.003 0l-.144-.084a2 2 0 00-2.735.733l-.22.383a2 2 0 00.73 2.73l.155.088a2 2 0 01.998 1.732v.51a2 2 0 01-.998 1.731l-.154.09a2 2 0 00-.732 2.73l.221.382a2 2 0 002.735.733l.144-.084a2 2 0 012.003 0l.435.252a2 2 0 01.999 1.731V20a2 2 0 002 2h.434a2 2 0 002-2v-.18a2 2 0 01.999-1.731l.435-.252a2 2 0 012.003 0l.144.084a2 2 0 002.735-.733l.22-.383a2 2 0 00-.73-2.73l-.155-.088a2 2 0 01-.998-1.732v-.51a2 2 0 01.998-1.731l.154-.09a2 2 0 00.732-2.73l-.221-.382a2 2 0 00-2.735-.733l-.144.084a2 2 0 01-2.003 0l-.435-.252a2 2 0 01-.999-1.731V4a2 2 0 00-2-2z"/>
+<path d="M15 12a3 3 0 11-6 0 3 3 0 016 0z"/>
```
Comment (ericfennis) [inline icons/settings.svg:12]:
```suggestion
  <path d="M12.22 2h-.44a2 2 0 00-2 2v.18a2 2 0 01-1 1.73l-.43.25a2 2 0 01-2 0l-.15-.08a2 2 0 00-2.73.73l-.22.38a2 2 0 00.73 2.73l.15.1a2 2 0 011 1.72v.51a2 2 0 01-1 1.74l-.15.09a2 2 0 00-.73 2.73l.22.38a2 2 0 002.73.73l.15-.08a2 2 0 012 0l.43.25a2 2 0 011 1.73V20a2 2 0 002 2h.44a2 2 0 002-2v-.18a2 2 0 011-1.73l.43-.25a2 2 0 012 0l.15.08a2 2 0 002.73-.73l.22-.39a2 2 0 00-.73-2.73l-.15-.08a2 2 0 01-1-1.74v-.5a2 2 0 011-1.74l.15-.09a2 2 0 00.73-2.73l-.22-.38a2 2 0 00-2.73-.73l-.15.08a2 2 0 01-2 0l-.43-.25a2 2 0 01-1-1.73V4a2 2 0 00-2-2z" />
  <circle cx="12" cy="12" r="3" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Code improvements

**Comment by delnyn** [comment]
> To decrease the size of first node, I tried to split into 4 `path`, it didn't work.

WDYM by "It didn't work"? What didn't work?

**Comment by mittalyashu** [comment]
@delnyn This is what I mean.

CleanShot 2021-05-06 at 08 53 17@2x

Split into 4 different `path`.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**